i18n/en.toml

```diff
@@ -124,3 +124,9 @@ other = "Liability Notice"
 
 [search]
 other = "Search"
+
+[minute]
+other = "minute"
+
+[minutes]
+other = "minutes"
```

Comment:
We don't need to define two variables for minute. You can just do:
```
[minute]
one = "minute"
other = "minutes"
```
This will automatically use `minute` where singular is used and `minutes` where plural is used.